skore/src/skore/sklearn/_comparison/report.py

```diff
@@ -267,6 +268,88 @@ def cache_predictions(
             )
             progress.update(main_task, advance=1, refresh=True)
 
+    def get_predictions(
+        self,
+        *,
+        data_source: Literal["train", "test", "X_y"],
+        response_method: str,
```

Comment:
```suggestion
        response_method: Union[Literal["predict", "predict_proba", "decision_function"], str],
```

Reply:
```suggestion
        response_method: Literal["predict", "predict_proba", "decision_function"],
```
since there cannot be another choice